src/Microsoft.OData.Core/UriParser/Binders/MetadataBindingUtils.cs

```diff
@@ -64,7 +64,7 @@ internal static SingleValueNode ConvertToTypeIfNeeded(SingleValueNode source, IE
                 {
                     string memberName = constantNode.Value.ToString();
                     IEdmEnumType enumType = targetTypeReference.Definition as IEdmEnumType;
-                    if(enumType.ContainsMember(memberName, StringComparison.Ordinal))
+                    if (enumType.ContainsMember(memberName, StringComparison.Ordinal))

```

Comment:
Should that honor the resolver's preference for case-sensitivity?

Reply:
This is not related to this PR. I'd prefer to leave it as it now and fix it if we do meet the problem.